### PR TITLE
Added a bash script which downloads file from s3 and upload them to ftp

### PR DESCRIPTION
**Pull request by beono**
I'm not an expert in bash, so feel free to give me some ideas to improve this script

### CODE REVIEW

File: ftp_uploader.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+	USAGE:
+		./ftp_uploader.sh "FTP_RECEIVER_IP" "prefix" "competition name"
+
+	EXAMPLE:
+		./ftp_uploader.sh "127.0.0.1" "backup/standings/scoreradar/stats/" "Mexico.CopaMX"
+
+'
+
+USER=erakhimberdin
```
Comment (makarski) [inline ftp_uploader.sh]: user?

File: ftp_uploader.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	echo "1) Downloading files from s3" && \
+	rm -f $DOWNLOAD_DIR/*.xml && \
+	go run main.go -bucket=motain-feeds -dir=$DOWNLOAD_DIR -prefix=$2 -regexp=$3 && \
+	read -p "Do you want to upload these files to the ftp_receiver server? (y/n): " -r
+	if [[ "$REPLY" = "y" ]]
+	then
+		echo "2) Uploading files ssh $USER@$FTP_RECEIVER_IP" && \
+		rsync -v -r $DOWNLOAD_DIR/*.xml $USER@$FTP_RECEIVER_IP:/home/$USER/tmp_feeds && \
+		echo "3) Moving files to $WATCHING_FOLDER"  && \
+		ssh $USER@$FTP_RECEIVER_IP $COMMAND
```
Comment (makarski) [inline ftp_uploader.sh]: user here as well.

File: ftp_uploader.sh [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+'
+
+USER=erakhimberdin
+WATCHING_FOLDER=/sharedstorage/middleware/pub/push/scoreradar/stats
+ILIGAHOME=/home/iliga
+DOWNLOAD_DIR=./tmp/s3feeds
+
+COMMAND="sudo mkdir -p $ILIGAHOME/tmp_feeds && \
+		sudo mv /home/$USER/tmp_feeds/*.xml $ILIGAHOME/tmp_feeds && \
```
Comment (makarski) [inline ftp_uploader.sh]: and here

File: ftp_uploader.sh [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+else
+	FTP_RECEIVER_IP=$1
+	echo "1) Downloading files from s3" && \
+	rm -f $DOWNLOAD_DIR/*.xml && \
+	go run main.go -bucket=motain-feeds -dir=$DOWNLOAD_DIR -prefix=$2 -regexp=$3 && \
+	read -p "Do you want to upload these files to the ftp_receiver server? (y/n): " -r
+	if [[ "$REPLY" = "y" ]]
+	then
+		echo "2) Uploading files ssh $USER@$FTP_RECEIVER_IP" && \
+		rsync -v -r $DOWNLOAD_DIR/*.xml $USER@$FTP_RECEIVER_IP:/home/$USER/tmp_feeds && \
```
Comment (makarski) [inline ftp_uploader.sh]: and here.

**Comment by beono** [comment]
@makarski Fixed $USER

**Comment by beono** [comment]
#10

**Pull request merged**